community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacade.java

```diff
@@ -196,7 +197,7 @@ public GraphDatabaseFacade()
     /**
      * Create a new Core API facade, backed by the given SPI.
      */
-    public void init( SPI spi, Config config )
+    public void init( SPI spi, ThreadToStatementContextBridge txBridge, Config config )
```

Comment:
Hm.. what do you think about passing dependency resolver here instead of already resolved txBridge?
So we can make it thing more generic